src/explanations/pancakeswap.yaml

```diff
@@ -0,0 +1,113 @@
+
+    ### Binance Smart Chain
+
+
+    Binance Smart Chain (BSC) is a blockchain network developed by the cryptocurrency exchange Binance. It is compatible with the Ethereum Virtual Machine (EVM), which means that it can run smart contracts and decentralized applications (DApps) that are built on Ethereum.
+
+
+    BSC was designed to offer the benefits of Ethereum's smart contract functionality, while also providing faster transaction speeds and lower fees. It uses a proof-of-staked-authority (PoSA) consensus mechanism, which means that block validation is performed by a group of validators who have staked a certain amount of the Binance Coin (BNB) cryptocurrency.
+
+
+    BSC has its own native cryptocurrency, called Binance Coin (BNB), which is used to pay transaction fees on the network. It also supports a wide range of other cryptocurrency assets, including those built on Ethereum and other networks.
+
+    ### The Exchange
+
+
+    Pancakeswap allows users to buy, sell, and trade a wide range of cryptocurrency assets, including popular coins like Bitcoin and Ethereum, as well as lesser-known altcoins and NFTs. Users can view the available assets and their prices on the Pancakeswap exchange, and can place orders to buy or sell using the AMM algorithm.
+
+
+    One of the key features of Pancakeswap is its liquidity pools, which allow users to provide liquidity to the exchange in exchange for a share of the fees generated by the trades. This helps to ensure that there is always a healthy supply of assets available for trading and allows users to earn a passive income by providing liquidity.
+
+
+    ### Yield Farming
+
+
+    Yield farming on PancakeSwap is a way for users to earn the platform's native token, CAKE, by providing liquidity to the DEX's liquidity pools and staking liquidity provider (LP) tokens in yield farms. When a user provides liquidity to a liquidity pool, they receive LP tokens that represent their share of the pool. These LP tokens can then be staked in a yield farm to earn CAKE rewards.
+
+
+    The annual percentage rate (APR) earned through yield farming on PancakeSwap is made up of two components: the LP rewards APR earned through providing liquidity, and the farm base rewards APR earned by staking LP tokens in the farm. By staking LP tokens in a farm, users can earn CAKE rewards while still maintaining a position in their other tokens.
+
+
+    It is important to note that yield farming on PancakeSwap with a risk of impermanent loss. This occurs when the value of the tokens in the liquidity pool changes over time, leading to a difference between the value of the tokens when they were added to the pool and their value when they are withdrawn. This can result in a loss for the liquidity provider, even if the value of the tokens they received as a reward has increased.
+
+
+    ### Syrup Pools 
```

Comment:
Syrup pools heading should be before yield farm and the first step of yield farm is adding liquidity in syrup pools